src/main/java/org/jabref/gui/importer/fetcher/WebSearchPaneViewModel.java

```diff
@@ -174,3 +179,14 @@ public ValidationStatus queryValidationStatus() {
         return searchQueryValidator.getValidationStatus();
     }
 }
+
+class CompositeSearchFirstComparator implements Comparator<SearchBasedFetcher> {
```

Comment:
Not necessary

Reply:
Thank you for your review! The issue says "Search Selected" should be positioned at the beginning of the drop down, and this is the reason I implemented another Set and another Comparator to re-order the set. Should I use another way to implement this, or is there no need to position it? Thank you.

Reply:
Then just replace the original comparator with yours